Multisig/UI/Safe Management/Create Safe/SafeCreatingViewController.swift

```diff
@@ -44,30 +46,25 @@ class SafeCreatingViewController: UIViewController {
                                                object: nil)
 
         NotificationCenter.default.addObserver(self,
-                                               selector: #selector(safeCreated),
+                                               selector: #selector(safeSubmitted),
                                                name: .safeCreationUpdate,
                                                object: nil)
-        // TODO: bind the tx status changes to SafeCreatingViewController and remove times I defined there
-        Timer.scheduledTimer(withTimeInterval: 3, repeats: false) {_ in
-            NotificationCenter.default.post(name: .safeAccountOwnerCreated, object: nil)
-            Timer.scheduledTimer(withTimeInterval: 3, repeats: false) {_ in
-                NotificationCenter.default.post(name: .safeAccountOwnerSecured, object: nil)
-                Timer.scheduledTimer(withTimeInterval: 3, repeats: false) {_ in
-                    NotificationCenter.default.post(name: .safeCreationUpdate, object: nil)
-                }
-            }
-        }
+        onViewDidLoad()
     }
 
     @objc func accountCreated() {
         infoView1.set(status: .success)
+        // Fake delay until account is secured...
```

Comment:
what about this delay? Is it necessary?

Reply:
I have no idea, what this message is supposed to indicate. So I do not know where to emit this notification. That is why there is this delay. We should probably get rid of this bullet point.

Reply:
please describe why behind it. Current comment only creates confusion.